dashboard/src/components/AppView/AppView.tsx

```diff
@@ -240,6 +273,50 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
   private deleteApp = (purge: boolean) => {
     return this.props.deleteApp(this.props.releaseName, this.props.namespace, purge);
   };
+
+  private filterByKind = (kind: string) => {
+    const kubeItems = _.pickBy(this.props.resources, r => {
```

Comment:
any reason to use lodash here instead of `Array.filter` and `Array.forEach`?

Reply:
`resources` is an object not an array so we would need to use `Object.keys` + `Array.filter` + populating a new object, using `pickBy` is faster/more readable.